packages/core/util/Util.ts

```diff
@@ -43,6 +43,14 @@ export class Util {
 		return App.container.get<IFileSystem>(FS_TOKEN).fileExists(filePath);
 	}
 
+	public static readFile(filePath) {
```

Comment:
👀 Okay, this is a bit confusing. Why the two specific shortcuts, when the setup is intended for all usage to just grab the fs from the container and use it as needed?